test/metabase/api/card_test.clj

```diff
@@ -818,16 +817,21 @@
                        :field_ref     [:aggregation 0]}]
                      (t2/select-one-fn :result_metadata :model/Card :name card-name))))))))))
 
+(defn- updating-card-updates-metadata-query []
+  (mt/mbql-query venues {:fields [$id $name]}))
+
+(def ^:private ^{:arglists '([s])} norm (comp u/upper-case-en :name))
```

Comment:
If it deserves an `arglists` then `defn` is probably clearer.

Reply:
fixed